meinberlin/react/contrib/ControlBar.jsx

```diff
@@ -24,110 +25,127 @@ const getResultCountText = (count) => {
-                onClickToggleFilter={handleToggleFilters}
-                btnString={translated.filters}
-                showFiltersString={translated.showFilters}
-                hideFiltersString={translated.hideFilters}
-              />
-            </div>
+      <div className="control-bar">
+        <div className="flexgrid grid--3 control-bar__search">
+          <div className="span2">
+            <ControlBarSearch onSearch={(value) => handleSearch(value)} term={term} />
           </div>
-          {props.filters && expandFilters && (
-            <>
-              <div className="control-bar">
-                {Object.keys(props.filters).map((type, idx) => {
-                  const filterItem = props.filters[type]
-                  return (
-                    type !== 'ordering' && (
-                      <ControlBarDropdown
-                        key={'filter_' + idx}
-                        filter={filterItem}
-                        current={queryParams.get(type)}
-                        filterId={'id_' + type}
-                        onSelectFilter={(choice) => applyFilter(type, choice)}
-                      />
-                    )
-                  )
-                })}
-              </div>
-            </>
+          {props.filters?.ordering && (
+            <ControlBarDropdown
+              key="ordering_dropdown"
+              filter={props.filters.ordering}
+              current={queryParams.get('ordering')}
+              filterId="id_ordering"
+              onSelectFilter={(choice) => handleFilterChange('ordering', choice[0])}
+            />
           )}
-        </>
-        {/* only show result string if list filtered or searched not just ordered */}
-        {resultString && (
-          <div className="control-bar">
-            {props.numOfResults >= 0 && getResultCountText(props.numOfResults)}
-          </div>
-        )}
-        {term && (
-          <ControlBarSearchTerm term={term} onDismiss={() => applySearch('')} />
-        )}
+        </div>
+        <h3>{translated.filters}</h3>
+        <div className="flexgrid grid--3">
+          {filters.length > 0 && (
+            filters
+              .filter((f) => f.type !== 'ordering')
+              .slice(0, expandFilters ? filters.length : 3).map((filter) => (
+                <ControlBarDropdown
+                  key={filter.type}
+                  filter={filter}
+                  current={filter.value}
+                  filterId={'id_' + filter.type}
+                  onSelectFilter={(choice) => handleFilterChange(filter.type, choice[0])}
+                />
+              ))
+          )}
+        </div>
+        <button onClick={() => setExpandFilters(!expandFilters)} className="control-bar__filter-btn">
+          <i className={'fa fa-chevron-' + (expandFilters ? 'up' : 'down')} aria-hidden="true" />
+          {expandFilters ? translated.hideFilters : translated.showFilters}
+        </button>
```

Comment:
this is shown when there are no additional filters to be displayed so if we only have `Status`and `Archived`it is uneeded